lib/recurly/invoice.php

```diff
@@ -59,7 +59,7 @@ public static function getInvoicePdf($invoiceNumber, $locale = null, $client = n
    * @param string Unique account code
    * @param array additional invoice attributes (see writeableAttributes)
    * @return Recurly_InvoiceCollection collection of invoices on success
-   */
+   **/
```

Comment:
Doesn't really matter but the PHP docs only require the `/**` you can close with a single `*/`